app/jobs/google/search_progress_job.rb

```diff
@@ -9,10 +9,20 @@ def perform(user_id)
 
       raise ArgumentError unless user
 
-      SearchProgressChannel.broadcast_replace_to "toast_search_progress_stream:user_#{user.id}",
-                                                 target: 'toast_search_progress',
-                                                 partial: 'keywords/toast_search_progress',
-                                                 locals: { pending_count: user.keywords.pending.count }
+      SearchProgressChannel.broadcast_append_to "card_search_progress_stream:user_#{user.id}",
+                                                target: 'card_search_progress',
+                                                partial: 'keywords/card_search_progress',
+                                                locals: { presenter: presenter(user.keywords) }
+    end
+
+    private
+
+    def presenter(keywords)
+      Google::CardSearchProgressPresenter.new(
```

Comment:
How about moving this presenter in the `app/presenters` folder so all presenters stay in the same place. Also It will make `SearchProgressJob` lightweight